.pre-commit-config.yaml

```diff
@@ -42,12 +45,12 @@ repos:
   #   rev: v1.6.0
   #   hooks:
   #     - id: mypy
-  # - repo: local
-  #   hooks:
-  #     - id: pylint
-  #       name: pylint
-  #       entry: bash -c 'test -d .venv && . .venv/bin/activate ; pylint ${CI:+--reports=yes} "$@"' -
-  #       language: system
-  #       types: [ python ]
-  #       args:
-  #         - --disable=R,C
+  - repo: local
+    hooks:
+      - id: pylint
+        name: pylint
+        entry: bash -c 'test -d .venv && . .venv/bin/activate ; pylint ${CI:+--reports=yes} "$@"' -
+        language: system
+        types: [ python ]
+        exclude: "tests/input/"
```

Comment:
Can we store this in the "general" pylint config? Probably no CI should mess with our inputs

Reply:
Done in latest commit